### ISSUE TITLE
how to cancel async ?

### ISSUE THREAD

**Issue by sonygod** [haxetink/tink_await#37]
how to cancel async ? I can't find any keyword about cancel like

https://docs.microsoft.com/en-us/dotnet/standard/parallel-programming/task-cancellation?redirectedfrom=MSDN

**Comment by kevinresol** [comment]
I don't think it is possible to perform a cancellation in tink_await.

With plain tink_core you can implement with a `CallbackLink`

**Comment by sonygod** [comment]
@kevinresol I agree,but how about consider something  like  timeout  in asynchronous?
